source/documentation/tools/create-a-derived-table/dbt-athena-upgrade-guidance.md

```diff
@@ -1,6 +1,6 @@
 # dbt-athena Upgrade Guidance
 
-We are in the process of migrating from our in-house maintained fork of the `dbt-athena` adapter to the community maintained fork [dbt-athena-community](https://pypi.org/project/dbt-athena-community/) [recommended by dbt](https://docs.getdbt.com/reference/warehouse-setups/athena-setup). The guidance below details how you can test your models using the `dbt-athena-community` adapter. If you have any issues please get in touch via the [#ask-data-modelling](https://asdslack.slack.com/archives/C03J21VFHQ9) channel.
+We have now fully migrated to using [dbt-athena](https://pypi.org/project/dbt-athena-community/) [recommended by dbt](https://docs.getdbt.com/reference/warehouse-setups/athena-setup) as our interface with AWS. The guidance below details how you can test your models using during software updates to the `dbt-athena-community` adapter. If you have any issues please get in touch via the [#ask-data-modelling](https://asdslack.slack.com/archives/C03J21VFHQ9) channel.
```

Comment:
Is this a typo, tje use of "using during" doesnt quite read right?
> The guidance below details how you can test your models *using during software updates* to the `dbt-athena-community` adapter

But not too sure what it should be!